test/install_script/nvm_install_with_aliased_dot

```diff
@@ -1,13 +1,15 @@
 #!/bin/sh
 
 setup () {
+  shopt -s expand_aliases
   alias .='cat'
-  NVM_ENV=testing \. ../../install.sh
+  \. ../../install.sh > /dev/null
```

Comment:
the `NVM_ENV=testing` is still required tho; otherwise it will actually invoke the install script.

Reply:
But when I set `NVM_ENV=testing`, `nvm_do_install` is not executed, so I can’t test the sourcing.

Reply:
You can test the sourcing without executing that function; for example, `type nvm_do_install` will only exit zero if it was sourced correctly.

Reply:
I want to test line 368 in `install.sh`. `type nvm_do_install` is not enough. Or do you have another idea?

Reply:
ah, true. but in that case, you can invoke `nvm_do_install` manually.

Reply:
I think I'm done.